python/pyspark/logger/logger.py

```diff
@@ -66,10 +67,21 @@ def format(self, record: logging.LogRecord) -> str:
         }
         if record.exc_info:
             exc_type, exc_value, exc_tb = record.exc_info
+            stacktrace = traceback.extract_tb(exc_tb)
+
+            structured_stacktrace = [
+                {
+                    "class": exc_type.__name__ if exc_type else "UnknownException",
```

Comment:
This should be the name of the class in each stack frame, instead of the `exc_type.__name__`, but seems like the `frame` object doesn't contain the class name. As `exc_type.__name__` is already recorded as `exception.class`, I guess we can leave it as `None` here if it's not available?

Reply:
Yes, it is ok to ignore or left it as none if it's not available

Reply:
Makes sense. Just applied the suggestion. Thanks!